### PR TITLE
feat(plugin): add size in meters to point icon field component 

### PR DESCRIPTION
**Pull request by nina992**
# Overview 
add enable size in meter to point icon field component 

## Design link 
https://www.figma.com/file/xWM8xrtIAh5oMj0seLd9Jd/view-2.0?node-id=2414%3A80403&t=veBJ5HBiH2YUIpty-0

## Memo
https://github.com/reearth/reearth-web/pull/511

### CODE REVIEW

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointIcon.tsx [hunk 1]
```diff
@@ -11,6 +12,7 @@ const PointIcon: React.FC<BaseFieldProps<"pointIcon">> = ({
 }) => {
   const [imageURL, setImageURL] = useState(value.url ?? "");
   const [imageSize, setImageSize] = useState(value.size);
+  const [imageSizeInMeters, setImageSizeInMeters] = useState(value.imageSizeInMeters ?? true);
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointIcon.tsx:15]: Unless @lavalse or @rot1024 said it should default to true, I feel like this should default to false, so `!!value.imageSizeInMeters` works here

Comment (KaWaite) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointIcon.tsx:15]: Also, imageSizeInMeters is the API's field. For our purposes here in saving the value, simplifying is preferable. So could you make it value.sizeInMeters

Comment (nina992) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointIcon.tsx]: done

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointIcon.tsx [hunk 2]
```diff
@@ -21,20 +23,37 @@ const PointIcon: React.FC<BaseFieldProps<"pointIcon">> = ({
...
+        imageSizeInMeters === value.sizeInMeters)
+    )
+      return;
     const timer = setTimeout(() => {
       onUpdate({
         ...value,
         url: imageURL,
         size: imageSize,
-        override: { marker: { style: "image", image: imageURL, imageSize } },
+        override: {
```
[the remark this answers is not in the document]
Comment (nina992) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointIcon.tsx:43]: I though that we should pass the same name in API , done

**Review by KaWaite — APPROVED** [review]

**Pull request merged**